### PR TITLE
DCOS-57643: feat(servicetable): add endpoints link to action menu

### PR DESCRIPTION
**Pull request by Poltergeist**
This adds a link to the endpoints tab to the action menu of sdk's pods and apps.

CLOSES DCOS-57643

## Testing

Visit the Service Table
Check if the action menu of a service(SDK, app or/and pod) is containing "View Endpoints"
Click the Action the Endpoints tab should be shown now.

## Trade-offs

N/A
## Dependencies

N/A

## Screenshots

image

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -4327,7 +4327,7 @@
     },
     "bl": {
       "version": "1.2.2",
-      "resolved": "http://registry.npmjs.org/bl/-/bl-1.2.2.tgz",
+      "resolved": "https://registry.npmjs.org/bl/-/bl-1.2.2.tgz",
```
Comment (GeorgiSTodorov) [inline package-lock.json]: Why do we have package-lock changes here?

**Comment by Poltergeist** [comment]
@GeorgiSTodorov I wouldn't, I mean the endpoints tab is on the detail page so you either could click directly on the tab or click twice via the action menu. I would not add it there. Although I am happy to hear arguments against my opinion.

**Comment by GeorgiSTodorov** [comment]
I think it's good to have it for consistency between the two actions menus. @izharbuendia , what do you think?

**Comment by mperrotti** [comment]
I agree with Philipp about the menu, but I defer to Izhar on this one

**Comment by GeorgiSTodorov** [comment]
Failing system test seems unrelated.

**Pull request merged**